app/src/main/java/endava/codebase/android/movieapp/ui/component/ProgressBar.kt

```diff
@@ -0,0 +1,62 @@
+package endava.codebase.android.movieapp.ui.component
+
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Text
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+private const val DEGREES = 360f
+private const val SWEEP_START = 90f
+
+@Composable
+fun ProgressBar(movieProgress: Float, modifier: Modifier) {
+    Box(modifier = modifier.wrapContentSize()) {
+        Canvas(
+            modifier = modifier
+                .size(60.dp)
+                .padding(5.dp)
+        ) {
+            drawArc(
+                color = Color(100, 340, 100),
```

Comment:
You can extract colors to util files, there should already be some colors defined.